### PR TITLE
Show funding information per work. 

### PR DESCRIPTION
**Pull request by mfenner**
## Purpose
Show funding information for a specific work, and link to funder organization page in `akita` if a Crossref Funder ID is included. This works for both Crossref and DataCite DOIs and implements bi-directional linking (funder => works, and work => funder).

closes: #120

## Approach
Use `WorkFunding` component. Link to funding organization using the Crossref Funder ID and the support for external organizational identifiers implemented in #118.

#### Open Questions and Pre-Merge TODOs
- [ ] look into providing custom award URLs for awards from important funders.

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)

## Reviewer, please remember our [guidelines](https://datacite.atlassian.net/wiki/spaces/TEC/pages/1168375809/Pull+Request+Guidelines):

- Be humble in the language and feedback you give, ask don't tell.
- Consider using positive language as opposed to neutral when offering feedback. This is to avoid the negative bias that can occur with neutral language appearing negative.
- Offer suggestions on how to improve code e.g. simplification or expanding clarity.
- Ensure you give reasons for the changes you are proposing.

Bildschirmfoto 2020-09-03 um 12 41 48

### CODE REVIEW

File: src/components/Work/Work.tsx [hunk 1]
```diff
@@ -200,6 +205,16 @@ const DoiPresentation: React.FunctionComponent<Props> = ({ doi }) => {
       <WorkMetadata metadata={doi} linkToExternal={true}></WorkMetadata>
       {exportMetadata()}
       {formattedCitation()}
+      {showFunding && (
+        <h3 className="member-results" id="work-funding">Funding</h3>
+      )}
+      {showFunding && (
```
Comment (richardhallett) [inline src/components/Work/Work.tsx:211]: Minor style comment, no need to do check twice can just do it once for the whole thing right?

File: src/components/WorkFunding/WorkFunding.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+import { FundingReference } from '../WorkContainer/WorkContainer'
+import { doiFromUrl } from '../../utils/helpers'
+
+type Props = {
+  funding: FundingReference
+}
+
+const WorkFunding: React.FunctionComponent<Props> = ({ funding }) => {
+  console.log(funding)
```
Comment (richardhallett) [inline src/components/WorkFunding/WorkFunding.tsx:12]: Remove console.log

**Review by richardhallett — APPROVED** [review]
Minor comments but nothing major, approved.

File: src/components/Work/Work.tsx (hunk 1 above)
Comment (mfenner) [reply src/components/Work/Work.tsx:211]: I wasn't sure how to nest the code (first the condition then the loop) properly. Can look into it again.

**Pull request merged**